src/routes/boot.ts

```diff
@@ -65,7 +65,7 @@ export type Experimentation = {
 
 export type BaseBoot = {
   visit: { visitId: string; sessionId: string };
-  alerts: Omit<Alerts, 'userId'>;
+  alerts: Omit<Alerts, 'userId' | 'flags'>;
```

Comment:
We have to make sure we don't select this column also

Reply:
Lee made it a [seperate](https://dailydotdev.atlassian.net/jira/software/projects/WT/boards/11?selectedIssue=WT-1924) ticket, if thats ok with you?

Reply:
The ticket refers to the mutation to update alerts. I talk about the query that fetches alerts from the database for boot. If we introduce it in a different ticket it means there will be a period of time that users get this information even though they shouldn't

Reply:
@idoshamun I added a function for now, let me know what you think.